include/pingcap/pd/Client.h

```diff
@@ -64,6 +65,18 @@ class Client : public IClient
 
     std::string getLeaderUrl() override;
 
+    // ResourceControl related.
+    resource_manager::ListResourceGroupsResponse listResourceGroups(const resource_manager::ListResourceGroupsRequest &) override;
+
+    resource_manager::GetResourceGroupResponse getResourceGroup(const resource_manager::GetResourceGroupRequest &) override;
+
+    resource_manager::PutResourceGroupResponse addResourceGroup(const resource_manager::PutResourceGroupRequest &) override;
```

Comment:
`PutResourceGroupResponse` is inconsistent with `addResourceGroup`.

Reply:
The original RPC definition use PutXXX for `addResourceGroup`. Check: https://github.com/pingcap/kvproto/blob/master/proto/resource_manager.proto#L17